src/main/groovy/io/cify/runner/utils/CapabilityParser.groovy

```diff
@@ -10,33 +10,46 @@ import org.apache.logging.log4j.core.Logger
  *
  * Created by FOB Solutions
  */
-
 class CapabilityParser {
+    /**
+     * Capabilities parsing strategy, to put all capabilities into one {@link Capabilities} object
+     */
+    public static final String STRATEGY_ALL_IN_ONE = "all_in_one"
+    /**
+     * Capabilities parsing strategy, to put each capability into a separate {@link Capabilities} object
+     */
+    public static final String STRATEGY_ONE_BY_ONE = "one_by_one"
+    /**
+     * Capabilities parsing strategy, that creates all possible variations of capabilities with one capability
+     * of each kind (browser, android, ios) per {@link Capabilities} object
+     */
+    public static final String STRATEGY_VARIATIONS = "variations"
 
     private static final Logger LOG = LogManager.getLogger(this.class) as Logger
 
     private static final String ANDROID = "android"
     private static final String IOS = "ios"
     private static final String BROWSER = "browser"
-    private static final String SET = "set"
     private static final String DEFAULTS = "defaults"
+    private static final String CAPABILITIES = "capabilities"
+    private static final String STRATEGY = "strategy"
 
     /**
      * Gets full capabilities list from file or default list
      *
      * @return list of capabilities
-     * */
+     */
     static List generateCapabilitiesList(String capabilitiesFilePath, String extraCapabilities, String farmUrl, String capabilitiesFromCmd) {
```

Comment:
Completely unclear from a first sight what this method returns. Would be good to have an explicit statement

Reply:
Yep, done